docs/install.rst

```diff
@@ -8,9 +8,9 @@ Install version 3 with :pip:ref:`pip install`::
 
     pip install django-autocomplete-light
 
-Or, install the dev version with git::
+Or, install the dev version from Github::
 
-    pip install -e git+https://github.com/yourlabs/django-autocomplete-light.git#egg=django-autocomplete-light
+    pip install https://github.com/yourlabs/django-autocomplete-light/archive/master.zip
```

Comment:
If you install the dev version this way then you can't develop in the package to contribute without re-installing every time you change a file.

Perhaps we could mention both ways of installing ?